src/Behat/Testwork/Cli/ConvertConfigCommand.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * This file is part of the Behat Testwork.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Testwork\Cli;
+
+use Behat\Config\Config;
+use Behat\Config\Converter\CustomPrettyPrinter;
+use Behat\Config\Converter\UseStatementAdder;
+use Behat\Testwork\ServiceContainer\Configuration\ConfigurationLoader;
+use Behat\Testwork\ServiceContainer\Exception\ConfigurationLoadingException;
+use PhpParser\BuilderFactory;
+use PhpParser\Node\Stmt\Nop;
+use PhpParser\Node\Stmt\Return_;
+use PhpParser\NodeTraverser;
+use Symfony\Component\Console\Command\Command as BaseCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Yaml\Yaml;
+
+final class ConvertConfigCommand extends BaseCommand
+{
+    private OutputInterface $output;
+
+    public function __construct(
+        private ConfigurationLoader $configurationLoader,
+    ) {
+        parent::__construct('convert-config');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->output = $output;
+        $configPath = $this->configurationLoader->getConfigurationFilePath();
+        if (str_ends_with($configPath, '.php')) {
+            throw new ConfigurationLoadingException(sprintf('Configuration file `%s` is already in PHP format', $configPath));
+        }
+
+        $this->output->writeln(['', 'Starting conversion']);
+        $this->convertFile($configPath);
+        $this->output->writeln(['Conversion finished', '']);
+        return 0;
+    }
+
+    private function convertFile(string $filePath): string
+    {
+        $this->output->writeln('Converting configuration file: ' . $filePath);
+
+        $yamlConfig = (array) Yaml::parse(file_get_contents($filePath));
```

Comment:
We load the yaml file and first of all convert any imported file by calling this function recursively